common/views/components/ResetActiveFilters/ResetActiveFilters.tsx

```diff
@@ -0,0 +1,294 @@
+  margin-left: 6px;
+  padding-top: 2px;
+`;
+
+type CancelFilterProps = {
+  text?: string;
+  children?: ReactNode;
+};
+
+const CancelFilter: FunctionComponent<CancelFilterProps> = ({
+  text,
+  children,
+}: CancelFilterProps): ReactElement<CancelFilterProps> => {
+  return (
+    <Space
+      as="span"
+      h={{
+        size: 'm',
+        properties: ['margin-right'],
+      }}
+    >
+      <Space
+        as="span"
+        h={{
+          size: 'xs',
+          properties: ['margin-right'],
+        }}
+      >
+        <Icon
+          name="cross"
+          extraClasses="icon--match-text icon--silver v-align-middle"
+        />
+      </Space>
+      <span className="visually-hidden">remove </span>
+      {text || children}
+    </Space>
+  );
+};
+
+export const ResetActiveFilters: FunctionComponent<ResetActiveFilters> = ({
+  workTypeFilters,
+  productionDatesFrom,
+  productionDatesTo,
+  worksRouteProps,
+  imagesColor,
+  workTypeInUrlArray,
+  aggregations,
+  resetFilters,
+  enableMoreFilters,
+  languagesInUrl,
+  subjectsInUrl,
+  genresInUrl,
+}: ResetActiveFilters) => {
+  const languagesFilters = aggregations?.languages?.buckets || [];
+  return (
+    <Space
+      v={{ size: 's', properties: ['padding-top'] }}
+      h={{ size: 'm', properties: ['padding-left', 'padding-right'] }}
+      className="tokens bg-white"
+    >
+      <div className={classNames({ [font('hnm', 5)]: true })}>
+        <div>
+          <h2 className="inline">
+            <Space
+              as="span"
+              h={{
+                size: 'm',
+                properties: ['margin-right'],
+              }}
+            >
+              Active filters:
+            </Space>
+          </h2>
+          {productionDatesFrom && (
+            <NextLink
+              passHref
+              {...worksLink(
+                {
+                  ...worksRouteProps,
+                  page: 1,
+                  productionDatesFrom: null,
+                },
+                'cancel_filter/production_dates_from'
+              )}
+            >
+              <a>
+                <CancelFilter text={`From ${productionDatesFrom}`} />
+              </a>
+            </NextLink>
+          )}
+          {productionDatesTo && (
+            <NextLink
+              passHref
+              {...worksLink(
+                {
+                  ...worksRouteProps,
+                  page: 1,
+                  productionDatesTo: null,
+                },
+                'cancel_filter/production_dates_to'
+              )}
+            >
+              <a>
+                <CancelFilter text={`To ${productionDatesTo}`} />
+              </a>
+            </NextLink>
+          )}
+          {imagesColor && (
+            <NextLink
+              passHref
+              {...imagesLink(
+                {
+                  ...worksRouteProps,
+                  page: 1,
+                  color: null,
+                  locationsLicense: null,
```

Comment:
Why does this clear the license too?

Reply:
Yeah this was there before which is moved into the ResetActiveFilters component. I think this was partially integrated from routing and API calls but not the UI for this.